### PR TITLE
Italian translation

### PR DESCRIPTION
**Pull request by amivaleo**
Hi!
I added the italian translation to your extension.
I hope you will appreciate it. :)

### CODE REVIEW

File: src/locale/it/it.po [hunk 1]
```diff
@@ -0,0 +1,193 @@
...
+# This file is distributed under the same license as the PACKAGE package.
+# FIRST AUTHOR <EMAIL@ADDRESS>, YEAR.
+#
+msgid ""
+msgstr ""
+"Project-Id-Version: \n"
+"Report-Msgid-Bugs-To: \n"
+"POT-Creation-Date: 2017-04-19 12:23+0200\n"
+"PO-Revision-Date: 2017-04-20 09:43+0200\n"
+"Last-Translator: Raphael Rochet <raphael@rri.fr>\n"
```
Comment (OttoAllmendinger) [inline src/locale/it/it.po]: Feel free to add your name here :)

File: src/locale/it/it.po [hunk 2]
```diff
@@ -0,0 +1,193 @@
...
+"Language-Team: \n"
+"Language: fr\n"
```
Comment (OttoAllmendinger) [inline src/locale/it/it.po]: `Language: it`

File: src/locale/it/it.po [hunk 3]
```diff
@@ -0,0 +1,193 @@
...
+msgstr "Scorciatoie"
+
+#: src/prefs.js:81
+msgid "Show indicator"
+msgstr "Mostra l'indicatore"
+
+# FIXME this is translation of "Default Click Action"
+#: src/prefs.js:107
+msgid "Primary Button"
+msgstr "Comportamento al click"
```
Comment (OttoAllmendinger) [inline src/locale/it/it.po]: I think this is still the italian translation of "Default click action" or "Click action". Can you change it to "Primary Button"?

File: src/locale/it/it.po [hunk 4]
```diff
@@ -0,0 +1,193 @@
...
+"X-Generator: Poedit 1.8.7.1\n"
+"Plural-Forms: nplurals=2; plural=(n > 1);\n"
+
+#: src/filename.js:27
+msgid "Screenshot"
+msgstr "Screenshot"
+
+#: src/filename.js:27
+msgid "Screenshot (literal)"
+msgstr "Screenshot"
```
Comment (OttoAllmendinger) [inline src/locale/it/it.po]: `msgstr "\"Screenshot\""` is better

File: src/locale/it/it.po (hunk 1 above)
Comment (amivaleo) [inline src/locale/it/it.po]: Done. :)

File: src/locale/it/it.po (hunk 2 above)
Comment (amivaleo) [inline src/locale/it/it.po]: Fixed.

File: src/locale/it/it.po (hunk 4 above)
Comment (amivaleo) [inline src/locale/it/it.po]: Yep. Done.

File: src/locale/it/it.po (hunk 3 above)
Comment (amivaleo) [inline src/locale/it/it.po]: Ok, I literally translated that now.
Primary Button -> Pulsante primario

**Comment by amivaleo** [comment]
I updated the ,po file according to your suggestions, and also deleted the old .mo file and uploaded the new one.
I don't know if I have to make another pull request now or if you can already see the changes. I'm still learning how to use github.

**Comment by OttoAllmendinger** [comment]
You did it correctly, you can just push new commits to the branch to change the PR.

**Pull request merged**